setup.py

```diff
@@ -36,6 +37,14 @@
         "requests",
         "z3c.caching",
         "Zope",
+        "zope.annotation",
+        "zope.component",
+        "zope.event",
+        "zope.globalrequest",
+        "zope.i18nmessageid",
+        "zope.interface",
+        "zope.schema",
+        "zope.testing",
```

Comment:
That's not a mistake, it is not on the `test` requirements, I will not claim myself [clever than Hanno](https://github.com/plone/plone.cachepurging/commit/f2a9c617b002f107b269042f2ae18fb4361fee17), but maybe that somehow can be moved to the tests? 🤔

Reply:
I created an issue for this: #29.  For now, we can accept it in this PR.
In current plone coredev with just the latest release, I already see zope.testing in the instance script, so some other package is already pulling it in as well.